### ISSUE TITLE
Release 4.3.0 - Release Candidate 5 - Manual Testing - Prevent pending agents from keepping their state indefinitely

### ISSUE THREAD

**Issue by Rebits** [wazuh/wazuh-qa#2620]
## Release candidate info

|                                  |                                            |
|---------------------------------|--------------------------------------------|
| **Main release issue** | https://github.com/wazuh/wazuh/issues/12517|
| **Version** | 4.3.0                                  |
| **Release candidate #** | RC5                                         |
| **Tag** | https://github.com/wazuh/wazuh/tree/v4.3.0-rc5 |
| **Previous release candidate issue** | https://github.com/wazuh/wazuh/issues/12189 |
| **Next release candidate issue** | -                                          |

| Version | Description | PR |  QA test|
|--|--|--| --|
4.2.2 | This fix prevents pending agents from keeping their state indefinitely in the manager | https://github.com/wazuh/wazuh/pull/9042 | None

### Target

| OS | Install type |
|--|--|
| Linux |  Manager|

### Steps to reproduce

- Deploy manager/agent environment
- Configure the agent with the manager's IP
- Init the agent
- At the same time, the agent shows the pending status at the manager, stopping the agent using SIGKILL. This will stop suddelly the agent

```
pkill -9 wazuh

**Comment by Rebits** [comment]
### 04/03/2022 

### v4.2.1

```

[root@centos1 vagrant]# date
Fri Mar  4 10:42:09 UTC 2022
[root@centos1 vagrant]# cat /var/ossec/etc/ossec.conf | grep agents_disconnection_time
    <agents_disconnection_time>20s</agents_disconnection_time>
[root@centos1 vagrant]# /var/ossec/bin/agent_control -l

Wazuh agent_control. List of available agents:
   ID: 000, Name: centos1 (server), IP: 127.0.0.1, Active/Local
   ID: 006, Name: centos2, IP: any, Pending

List of agentless devices:

[root@centos1 vagrant]# date
Fri Mar  4 10:42:50 UTC 2022
[root@centos1 vagrant]# /var/ossec/bin/agent_control -l

Wazuh agent_control. List of available agents:
   ID: 000, Name: centos1 (server), IP: 127.0.0.1, Active/Local
   ID: 006, Name: centos2, IP: any, Pending

**Comment by Rebits** [comment]
### 07/03/2022 - RC5
This issue seems to be fixed in 4.3-RC5

https://user-images.githubusercontent.com/11089305/157010921-fd904384-b70e-4888-9bee-a17fb2286eb1.mp4
